AndroidBound/src/main/java/solutions/alterego/androidbound/android/ui/BindableRecyclerView.java

```diff
@@ -129,33 +147,69 @@ public BindableRecyclerView(Context context, @Nullable AttributeSet attrs, int d
 
         mItemTemplate = getItemTemplate(attrs);
         mTemplatesForObjects = new HashMap<>();
-        setLayoutManager(getLayoutManager(attrs));
+
+        processAttrs(attrs);
+        createLayoutManagerFromXmlParams();
+    }
+
+    public void setRtlLayout(boolean isRtlLayout) {
+        if (isRtlLayout != mRtlLayout) {
+            mRtlLayout = isRtlLayout;
+            createLayoutManagerFromXmlParams();
+        }
     }
 
     private int getItemTemplate(AttributeSet attrs) {
         return attrs.getAttributeResourceValue(null, BindingResources.attr.BindableListView.itemTemplate, 0);
     }
 
-    private LayoutManager getLayoutManager(AttributeSet attrs) {
-        LayoutManager layoutManager = new LinearLayoutManager(getContext(), OrientationHelper.VERTICAL, false); //default
+    private void processAttrs(AttributeSet attrs) {
+        if (attrs != null) {
+            mUseParentLayoutParams = attrs.getAttributeBooleanValue(null, BindingResources.attr.BindableRecyclerView.useParentLayoutParams, true);
+            mLayoutManagerOrientationString = attrs.getAttributeValue(null, BindingResources.attr.BindableRecyclerView.layoutManagerOrientation);
+            mRtlLayout = attrs.getAttributeBooleanValue(null, BindingResources.attr.BindableRecyclerView.layoutManagerReverse, mRtlLayout);
+            mSpanCount = attrs.getAttributeIntValue(null, BindingResources.attr.BindableRecyclerView.layoutManagerSpanCount, 1);
+            mInitialPrefetchCount = attrs.getAttributeIntValue(null, BindingResources.attr.BindableRecyclerView.initialPrefetchCount, 0);
+            mLayoutManagerType = attrs.getAttributeValue(null, BindingResources.attr.BindableRecyclerView.layoutManager);
+
+            Boolean nestedScrollingEnabled = attrs
```

Comment:
why an object and not the primitive type?

Reply:
no reason. changed it.